tasks/nexus_admin_password_poc1.secret

```diff
@@ -0,0 +1 @@
+43a3c5e9-538c-4acc-85f1-27a13594b2d3
```

Comment:
Do we need other environment equivalents of this file? I don't actually see where it's referenced in the role, and should it be in a bucket?

Reply:
Good spot, that file shouldn't be there. I've removed it. The secret it contained is no longer valid.